tree/ntuple/v7/src/RNTuple.cxx

```diff
@@ -101,6 +103,29 @@ std::string ROOT::Experimental::RNTupleReader::GetInfo(const ENTupleInfo what) {
    return "";
 }
 
+
+void ROOT::Experimental::RNTupleReader::Print(std::ostream &output, int width, unsigned int fNoFields) {
+    if(width < 30) {
+        std::cout << "The width is too small! Should be at least 30.\n";
+        return;
+    }
+    RPrintVisitor fPrintVisitor(output, width, fNoFields);
```

Comment:
`fPrintVisitor` is a local variable not a class member, should be called `printVisitor`.